### PR TITLE
Fixed article based on new Australia and Japan IP address (direct-routing-plan.md)

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7236.

@FilippSe could you kindly confirm this PR/information? Thanks!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -241,7 +241,10 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+>  As part of Teams Direct Routing expansion and service improvement, we have deployed new instances of Direct Routing infrastructure in Australia in November 2020. This is reflected in two additional IP addresses (52.114.16.74 and 52.114.20.29) to which following FQDNs will be resolved for Australian customers – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pstnhub.microsoft.com. You need to add these two IP addresses (52.114.16.74 and 52.114.20.29) to your IP Access Control Lists (ACLs) and open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signalling.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md:244]:
```suggestion
>  As part of Teams Direct Routing expansion and service improvement, we have deployed new instances of Direct Routing infrastructure in Australia in November 2020. This is reflected in two additional IP addresses (52.114.16.74 and 52.114.20.29) to which the following FQDNs will be resolved for Australian customers – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com, and sip3.pstnhub.microsoft.com. You need to add these two IP addresses (52.114.16.74 and 52.114.20.29) to your IP Access Control Lists (ACLs) and open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.
```

File: Teams/direct-routing-plan.md [hunk 2]
```diff
@@ -241,7 +241,10 @@ The FQDNs – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pst
...
+
+> [!IMPORTANT]
+>  As part of Teams Direct Routing expansion and service improvement, we have deployed new instances of Direct Routing infrastructure in Japan in May 2021. This is reflected in two additional IP addresses (52.114.36.156 and 52.114.32.169) to which following FQDNs will be resolved for Japan customers – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com and sip3.pstnhub.microsoft.com. You need to add these two IP addresses (52.114.36.156 and 52.114.32.169) to your IP Access Control Lists (ACLs) and open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signalling.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md:247]:
```suggestion
> As part of Teams Direct Routing expansion and service improvement, we have deployed new instances of Direct Routing infrastructure in Japan in May 2021. This is reflected in two additional IP addresses (52.114.36.156 and 52.114.32.169) to which the following FQDNs will be resolved for Japan customers – sip.pstnhub.microsoft.com, sip2.pstnhub.microsoft.com, and sip3.pstnhub.microsoft.com. You need to add these two IP addresses (52.114.36.156 and 52.114.32.169) to your IP Access Control Lists (ACLs) and open ports for all these IP addresses in your firewall to allow incoming and outgoing traffic to and from the addresses for signaling.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional suggestions, please (note, "signalling" is correct in UK English but not US English.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**